README.md

```diff
@@ -72,9 +72,13 @@ expect from the core classes from the ECE and CS programs at CMU.
 - [18-794: Pattern Recognition Theory](electives/18794.md)
 - [18-847F: Foundations of Cloud and Machine Learning Infrastructure](electives/18847F.md)
 - [18-898D: Graph Signal Processing and Geometric Learning](electives/18898D.md)
-- [24-104: Maker Series: Intro to Modern Making](electives/24104.md)
+
+## General Education Electives
 - [80-180: Nature of Language](electives/80180.md)
 
+## Others
```

Comment:
I think "Others" is a vague title and would make it harder for people to find this. Do we want something more specific like "Minis" or something else related to building stuff?